lib/web/fotorama/fotorama.js

```diff
@@ -2183,11 +2167,6 @@ fotoramaVersion = '4.6.4';
             if (o_allowFullScreen) {
                 $fullscreenIcon.prependTo($stage);
                 o_nativeFullScreen = FULLSCREEN && o_allowFullScreen === 'native';
-
-                // Due 300ms click delay on mobile devices
```

Comment:
This bug was fixed in iOS 11

Reply:
I'm not sure if we can remove it, as according to https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements_browsers.html Magento should support even iOS 7.

Reply:
The documentation is updated https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements_browsers.html